_data/simple-icons.json

```diff
@@ -55,6 +55,11 @@
             "hex": "0099E5",
             "source": "https://about.500px.com/press"
         },
+        {
+            "title": "9gag",
+            "hex": "000000",
+            "source": "https://drive.google.com/file/d/1-kG-D8X2he4YwnrW27jEGh4zysVrVMB5/"
```

Comment:
I would prefer to use the logo from https://de.wikipedia.org/wiki/Datei:9gag_logo.svg